browser/js/content.js

```diff
@@ -2,6 +2,8 @@
 // widget-rendering code into some page.
 "use strict";
 
+const bel = bundle.bel;
+const morphdom = bundle.morphdom;
```

Comment:
For the sake of keeping things simple, I'm keeping access to bundle stuff just inside content.js for now.